apps/condo/domains/banking/schema/BankIntegrationAccessRight.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Generated by `createschema banking.BankIntegrationAccessRight 'integration:Relationship:BankIntegration:PROTECT; user:Relationship:User:CASCADE;'`
+ */
+
+const { Relationship } = require('@keystonejs/fields')
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/banking/access/BankIntegrationAccessRight')
+const { SERVICE_USER_FIELD } = require('@condo/domains/miniapp/schema/fields/accessRight')
+
+
+const BankIntegrationAccessRight = new GQLListSchema('BankIntegrationAccessRight', {
+    schemaDoc: 'Link between bank integrations and users. The existence of the object means that there is user has access to integration',
```

Comment:
A good practice to write comments is not to repeat what is clear by reading code below, but to provide unobvious details, that does not explicitly visible in code.

For example,
"Link between bank integrations and users" — it's clear that this schema has associations, when you read the code;
"The existence of the object" its also clear, that when object will exist, user will get something.
So, to maximise value of this comment, — write about high-level logic, what is not directly visible in the code below, for example:

```suggestion
    schemaDoc: 'Gives ability to service-user to access all schema records, connected to specified integration',
```